### ISSUE TITLE
Migrate xrefs to a new system that allows to specify meanings of xrefs

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#20147]
This follows on #20135 

It would be clearer to users if we specified types of xrefs

SSFOM: Simple Shared Format for Ontology Mapping

**Comment by balhoff** [comment]
One possibility is that the new system involves direct use of properties like `skos:exactMatch`, `skos:narrowMatch`, etc. Examples can be seen [in MONDO](http://www.ontobee.org/ontology/MONDO?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0005015). A complication is that these properties are generally used with IRI values, not CURIE string literals (this expectation is in the SKOS spec).

As a stopgap, and as a simplification of the SKOS approach which we could later automatically generate, we can stick with `oboInOwl:hasDbXref` with literal values (e.g. "EC:3.4.24.14"). But we can annotate these annotations with a scope annotation.

Initially I thought we could point to the intended SKOS property as the value of the axiom annotation. However it seems that OBO format does not preserve IRI values in axiom annotations. For example, if this is entered in Protege:

<img width="960" alt="axiom_annotation" src="https://user-images.githubusercontent.com/210210/102123604-16a3c900-3e15-11eb-8408-2d99a01c925f.png">

the annotated xref is saved into OBO as:

```
[Term]
id: GO:0004222
name: metalloendopeptidase activity
xref: EC:3.4.24.14 {http://www.w3.org/2004/02/skos/core#scopeNote="narrowMatch"}
```

When re-opened in Protege, the value is just the string "narrowMatch".

**Comment by cmungall** [comment]
yes, in mondo we just use literals. When publishing for release we make skos triples.

**Comment by balhoff** [comment]
Here's an ODK ticket related to SSSOM exports: https://github.com/INCATools/ontology-development-kit/issues/106#issuecomment-750473354

**Comment by pgaudet** [comment]
see also https://arxiv.org/abs/2112.07051

**Comment by pgaudet** [comment]
Specs are here: https://mapping-commons.github.io/sssom/spec/
